File: 11875 Brick Game/solution.c
#include <stdio.h>

int main() {
    int T;
    scanf("%d", &T);
    for(int i = 1; i <= T; ++i) {
        int N;
        scanf("%d", &N);
        int ages[N];
        for(int j = 0; j < N; ++j)
            scanf("%d", &ages[j]);
        printf("Case %d: %d\n", i, ages[N >> 1]);
    }
    return 0;
}

File: 11875 Brick Game/solution.java
import java.io.*;

public class Main {
	public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StreamTokenizer st = new StreamTokenizer(br);
		st.nextToken();
        int T = (int)st.nval;
        StringBuilder output = new StringBuilder();
        for(int i = 1; i <= T; ++i) {
            st.nextToken();
            int N = (int)st.nval;
            int[] ages = new int[N];
            for(int j = 0; j < N; ++j) {
                st.nextToken();
                ages[j] = (int)st.nval;
            }
            output.append("Case ").append(i).append(": ").append(ages[N >> 1]).append('\n');
        }
	    System.out.print(output);
	}
}
